hack/upgrade-test.sh

```diff
@@ -226,7 +226,8 @@ ${CMD} wait deployment ${HCO_DEPLOYMENT_NAME} --for condition=Available -n ${HCO
 
 Msg "verify the hyperconverged-cluster deployment is using the new image"
 
-./hack/retry.sh 6 30 "${CMD} get deployments -n ${HCO_NAMESPACE} -o yaml | grep image | grep hyperconverged-cluster | grep ${REGISTRY_IMAGE_URL_PREFIX}"
+SEARCH_PHRASE=$(echo ${REGISTRY_IMAGE_URL_PREFIX} | sed -r "s|.*/([^/]*/stable).*|\1|")
```

Comment:
What about using the ${OPENSHIFT_BUILD_NAMESPACE} environment variable, which already contains that string (instead of extracting it using regex) ?

https://github.com/redhat-developer/devconsole-operator/tree/master/openshift-ci#devconsole-operator-registry